src/lodash-modules.js

```diff
@@ -1,35 +1,28 @@
 import path from 'path';
-import {includes, result, zipObject} from 'lodash';
+import {findKey, transform} from 'lodash';
 import fs from 'fs';
-import {aliasToReal} from 'lodash-cli/lib/mapping';
```

Comment:
Probably pedantic, but I did this originally for support of `npm modules`. I suppose this is better as `lodash-cli` is a pretty cumbersome download and we can't support the npm modules til https://github.com/babel/babel/issues/1833 anyway